Project.toml

```diff
@@ -24,7 +24,7 @@ AbstractAlgebra = "^0.11.1"
 Antic_jll = "~0.2.2"
 Arb_jll = "~2.18.1"
 BinaryProvider = "0.4, 0.5"
-FLINT_jll = "~2.6.3"
+FLINT_jll = "~200.690.0"
 LoadFlint = "^0.3.4"
```

Comment:
@thofma we also need to update the LoadFlint requirement here (and for that in turn, make a LoadFlint release).